subjects/fromto/README.md

```diff
@@ -0,0 +1,48 @@
+## fromto
+
+### Instructions
+
+Write a function that takes two `integers` and returns a `string` representing the range of numbers from the first to the second.
+
+- The function can return the numbers in any order.
```

Comment:
This line should not exist. Instructions are explicit enough, it prints in ascending or descending depending on order of `first` and `second` arguments

Reply:
Thank you, it is fixed.